RecoEgamma/EgammaHLTProducers/interface/HLTCaloObjInRegionsProducer.h

```diff
@@ -67,7 +67,7 @@ class EtaPhiRegion{
     ~EtaPhiRegion(){}
   bool operator()(float eta,float phi)const{
     return reco::deltaR2(eta,phi,centreEta_,centrePhi_)<maxDeltaR2_ ||
-      (std::abs(eta-centreEta_)<maxDEta_ && std::abs(reco::deltaPhi(phi,centrePhi_)<maxDPhi_));}
+      (std::abs(eta-centreEta_)<maxDEta_ && std::abs(reco::deltaPhi(phi,centrePhi_))<maxDPhi_);}
```

Comment:
Thanks for noticing.
RecoEgamma/EgammaHLTProducers is HLT only and HLT signed off.
But there is more like that in this PR in reco as well.